medusa/tv/series.py

```diff
@@ -2198,7 +2198,7 @@ def want_episode(self, season, episode, quality, forced_search=False,
             )
             return False
 
-        cur_status, cur_quality = int(sql_results[0]['status']), int(sql_results[0]['quality'])
+        cur_status, cur_quality = int(sql_results[0]['status'] or UNSET), int(sql_results[0]['quality'] or Quality.NA)
```

Comment:
This would allow `cur_status` to be `0`, skipping the proper status check (see first comment).